torch/utils/cpp_extension.py

```diff
@@ -2055,7 +2055,7 @@ def _get_cuda_arch_flags(cflags: Optional[list[str]] = None) -> list[str]:
 
     supported_arches = ['3.5', '3.7', '5.0', '5.2', '5.3', '6.0', '6.1', '6.2',
                         '7.0', '7.2', '7.5', '8.0', '8.6', '8.7', '8.9', '9.0', '9.0a',
-                        '10.0', '10.1', '12.0']
+                        '10.0', '10.0a', '10.1', '10.1a', '12.0', '12.0a']
```

Comment:
@drisspg already landing something like that

Reply:
I just closed mine, since this is adds the full family